packages/babel-standalone/package.json

```diff
@@ -0,0 +1,146 @@
+    "babel-plugin-transform-es2015-modules-umd": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-object-super": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-parameters": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-shorthand-properties": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-spread": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-sticky-regex": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-template-literals": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-typeof-symbol": "7.0.0-alpha.19",
+    "babel-plugin-transform-es2015-unicode-regex": "7.0.0-alpha.19",
+    "babel-plugin-transform-es3-member-expression-literals": "7.0.0-alpha.19",
+    "babel-plugin-transform-es3-property-literals": "7.0.0-alpha.19",
+    "babel-plugin-transform-es5-property-mutators": "7.0.0-alpha.19",
+    "babel-plugin-transform-eval": "7.0.0-alpha.19",
+    "babel-plugin-transform-exponentiation-operator": "7.0.0-alpha.19",
+    "babel-plugin-transform-export-extensions": "7.0.0-alpha.19",
+    "babel-plugin-transform-flow-comments": "7.0.0-alpha.19",
+    "babel-plugin-transform-flow-strip-types": "7.0.0-alpha.19",
+    "babel-plugin-transform-function-bind": "7.0.0-alpha.19",
+    "babel-plugin-transform-inline-environment-variables": "6.8.0",
+    "babel-plugin-transform-jscript": "7.0.0-alpha.19",
+    "babel-plugin-transform-member-expression-literals": "6.8.4",
+    "babel-plugin-transform-merge-sibling-variables": "6.8.5",
+    "babel-plugin-transform-minify-booleans": "6.8.2",
+    "babel-plugin-transform-node-env-inline": "6.8.0",
+    "babel-plugin-transform-object-assign": "7.0.0-alpha.19",
+    "babel-plugin-transform-object-rest-spread": "7.0.0-alpha.19",
+    "babel-plugin-transform-object-set-prototype-of-to-assign": "7.0.0-alpha.19",
+    "babel-plugin-transform-property-literals": "6.8.4",
+    "babel-plugin-transform-proto-to-assign": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-constant-elements": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-display-name": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-inline-elements": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-jsx": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-jsx-compat": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-jsx-self": "7.0.0-alpha.19",
+    "babel-plugin-transform-react-jsx-source": "7.0.0-alpha.19",
+    "babel-plugin-transform-regenerator": "7.0.0-alpha.19",
+    "babel-plugin-transform-remove-console": "6.8.4",
+    "babel-plugin-transform-remove-debugger": "6.8.4",
+    "babel-plugin-transform-runtime": "7.0.0-alpha.19",
+    "babel-plugin-transform-simplify-comparison-operators": "6.8.4",
+    "babel-plugin-transform-strict-mode": "7.0.0-alpha.19",
+    "babel-plugin-transform-undefined-to-void": "6.8.2",
+    "babel-plugin-undeclared-variables-check": "6.22.0",
+    "babel-preset-babili": "0.1.4",
+    "babel-preset-env": "^2.0.0-alpha.18",
```

Comment:
Oh yeah, good point. I copied this from babel-standalone's repo and forgot to fix the version numbers.